index.html

```diff
@@ -1644,6 +1644,141 @@ <h4>Background</h4>
+		the protocol, transport, or application used to facilitate it. This feature allows people
+		to easily switch communication contexts without losing any security guarantees, and provides
+		a resilient backbone for all internet communications. Moving away from traditional identifiers
+		such as phone numbers and email addresses can help to break down the siloed ecosystems that
+		prevent data portability and make it difficult to implement secure communication technology
+		more broadly into applications and services. Current approaches to secure communication
+		often have the following drawbacks:
+        </p>
+        <ul>
+          <li>they rely on commonly used identifiers which are not directly under user control;</li>
+          <li>they use a central intermediary or centralized components to facilitation interactions;</li>
+          <li>they require one or more parties to have a highly available web server;</li>
+          <li>they have siloed and non-interoperable security architectures tied to a single domain.</li>
+        </ul>
+        <p>
+          DIDs help to solve this issue by enabling a style of communication which is truly peer-to-peer and
+		messaging-based. Message-based systems decouple the relationship between producers of messages
+		and consumers of messages, creating an abstraction that puts the two on equal footing and allows
+		for many different kinds of devices with differing requirements to interface in a flexible
+		and technology-neutral way. DIDs allow people to simultaneously use the same identifier for
+		both traditional communications and transactions that require a level of cryptographic security,
+		as well as switch contexts without losing any security guarantees. These identifiers are not
+		rented or borrowed from centralized providers but are rather directly under user control.
+        </p>
+      </section>
+      <section id="messagingDescription">
+        <h4>Description</h4>
+        <p>
+          Samira is a humanitarian aid worker in Khartoum and wants to create connections with new people she
+		meets. She has a smartphone but doesn’t have any internet access in remote areas that she visits
+		in her line of work. Since she can’t use the internet, most of her preferred methods of communication
+		are unavailable. When she meets a new person she wishes to communicate with, she establishes a
+		direct secure connection between her device and theirs by having them scan a QR code to pass on her
+		DID and start sharing messages secured by the keys referenced in her DID Document, which is served from
+		her device. She can maintain these secure connections whether she has access to the internet or not.
+		Whenever Samira is working on a new campaign or wants to raise awareness about issues affecting her
+		local region, she is able to securely communicate with all of the relevant parties, regardless of
+		the device, platform, or network infrastructure they are using.
```

Comment:
```suggestion
          Samira is a humanitarian aid worker in Khartoum who wants to create connections with new people she
		meets. She has a smartphone but doesn’t have any internet access in remote areas that she visits
		in her line of work. Since she can’t use the internet, most of her preferred methods of communication
		are unavailable. When she meets a new person she wishes to communicate with, she establishes a
		direct secure connection between her device and theirs by having them scan a QR code to pass on her
		DID and start sharing messages secured by the keys referenced in her DID Document, which is served from
		her device. She can maintain these secure connections whether she has access to the internet or not.
		Whenever Samira is working on a new campaign or wants to raise awareness about issues affecting her
		local region, she is able to securely communicate with all relevant parties, regardless of
		the device, platform, or network infrastructure they are using.
```